cmd/update-main_test.go

```diff
@@ -66,11 +86,12 @@ func TestGetCurrentReleaseTime(t *testing.T) {
 		{minioVersion1, "", releaseTime1, nil},
 		{minioVersion1, minioBinaryPath2, releaseTime1, nil},
 		{minioVersion2, minioBinaryPath2, releaseTime2, nil},
+		{minioVersion2, minioBinaryPath3, releaseTime3, nil},
```

Comment:
Also I think we would need to add nonexistent absolute path for minio binary test cases.